### PR TITLE
Fix sqlalchemy class lookup

### PR DESCRIPTION
**Pull request by ebreton**
This is a one-line fix to avoid the crash of the application in certain edge cases, where you defined an SQLAlchemy relationship where one of the model is not yet imported at the time the application creates it initial data.

See Issue #28 for all the details

### CODE REVIEW

File: {{cookiecutter.project_slug}}/backend/app/app/db/init_db.py [hunk 1]
```diff
@@ -2,6 +2,8 @@
 from app.core import config
 from app.models.user import UserCreate
 
+from .base import *  # noqa
```
Comment (ebreton) [inline {{cookiecutter.project_slug}}/backend/app/app/db/init_db.py]: the `# noqa` here is important, others wise the autoformat tools (lint.sh) would remove this line

**Comment by ebreton** [comment]
I will do that tomorrow :ok_hand:

**Comment by ebreton** [comment]
Good morning @tiangolo , the `from app.db import base` works fine :notes: . Changed made, and comment added.

Let me know whether there is anything else. I was hesitating for instance to add a new SubItem db_models to get automated testing and avoid regression (as described in #28), but I didn't... it seemed like too much noise...

**Comment by tiangolo** [comment]
Awesome! Thanks a lot for your work :tada: :cake:

**Pull request merged**